src/Components/Blazor/BlazorExtension/src/Microsoft.VisualStudio.BlazorExtension.csproj

```diff
@@ -1,5 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <IsProductPackage>true</IsProductPackage>
+    <IsPackable>false</IsPackable>
+    <IsTestProject>false</IsTestProject>
+    <AddImplicitReferences>false</AddImplicitReferences>
```

Comment:
Did you every figure out why IsTestProject was set to true? This project isn't named 'Test' so it *shouldn't* be.

Reply:
yeah Pranav and I spent a while on this. It looks like something that only happens in the design-time build. I'm going to remove the two extra properties, since this all builds bind and just displays wierd in the solution explorer.